content/geoip/release-notes/2024.mdx

```diff
@@ -65,27 +65,43 @@ are also a minFraud user,
 
 We will begin using R2 presigned URLs for all database downloads in order to
 increase the security and reliability of our services.
-[Learn more about R2 presigned URLs in Cloudflare’s online documentation.](https://developers.cloudflare.com/r2/api/s3/presigned-urls/)
+
+This means that when you hit a URL to download a GeoIP or GeoLite database we
+will return a redirect and you should make sure that your downloading servers
+don't have any proxies or firewall settings that would block the host we are
+redirecting to.
+
+**What do I need to do?**
 
 MaxMind users who download databases should make sure that their servers can
 make HTTPS connections to the following hostname:
 
 - `mm-prod-geoip-databases.a2649acb697e2c09b632799562c076f2.r2.cloudflarestorage.com`
 
-The permalinks from the download page in your
-[account portal](https://www.maxmind.com/en/accounts/current/geoip/downloads)
-(login required) will not be changing. You will be redirected from those
-permalinks to the R2 presigned URLs.
+**If you use a direct-download method (most CSV downloaders):**
+
+The permalinks from the download page in your account portal will not be
+changing. You will be redirected from those permalinks to the R2 presigned URLs.
+
+You can test to ensure that your server is properly configured by getting a
+permalink from the account portal and using this permalink with the `curl` or
+`wget` examples in
```

Comment:
There is also a concern that whatever tool they are using for the actual download would not follow redirects. I'm not sure what we should say about that though. i.e., they would make a request to the original URL expecting the database to be there, and while the response says there is a redirect, the client may not support redirects or may not be configured to follow the redirect.